libafl/src/fuzzer/mod.rs

```diff
@@ -431,11 +445,12 @@ where
         &mut self,
         state: &mut S,
         manager: &mut EM,
-        input: I,
+        input: &I,
```

Comment:
Is it better to move the clone into this guy?

Reply:
i think yes. we should clone right before when it's necessary. that's why i changed all the places into &I instead.